spec/rails_helper.rb

```diff
@@ -128,7 +128,7 @@
 
   config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
-    FactoryBot.create :vat_rate, effective_date: Date.new(1960, 1, 1), rate_base_points: 1750
+    FactoryBot.create(:vat_rate, :for_2011_onward)
```

Comment:
Just out of interest, could you elaborate on the reason for requiring this VAT rate to be set for the suite? 🙇

Reply:
I did not put it in, but it does affect a lot of specs as fee hooks require it for calculating totals.

Reply:
😱  => 🏃 😅